### PR TITLE
They look all right now

### PR DESCRIPTION
**Pull request by optimumtact**
:cl: oranges
add: Nanotrasen is an all right kind of company
/:cl:

the black knight looks at his cut off hand and wonders about the pointless futility of a life where entropy is slowly counting down to the heat death of the univerise

### CODE REVIEW

**Comment by kevinz000** [comment]
meme pr

**Comment by optimumtact** [comment]
You're an all right fellow @kevinz000

**Comment by LaharlMontogmmery** [comment]
NOT ENOUGH TAGS YET

**Comment by francinum** [comment]
quality content.

**Comment by Jordie0608** [comment]
Sorry you must've meant this PR for https://github.com/lifeweb/lifeweb

**Comment by hexafluoride** [comment]
hi

**Comment by optimumtact** [comment]
h-h-hey

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -345,7 +345,8 @@
 						msg += "<a href='?src=\ref[src];hud=s;add_crime=1'>\[Add crime\]</a> "
 						msg += "<a href='?src=\ref[src];hud=s;view_comment=1'>\[View comment log\]</a> "
 						msg += "<a href='?src=\ref[src];hud=s;add_comment=1'>\[Add comment\]</a>\n"
-
+	if(!get_bodypart(ARM_LEFT) && !get_bodypart(LEG_LEFT))
+		msg += "[t_He] looks all right now\n"
```
Comment (Cheridan) [inline code/modules/mob/living/carbon/human/examine.dm]: need a period at the end!!1

**Comment by Jordie0608** [comment]
\>This was actually merged
Chelp

**Comment by optimumtact** [comment]
Tis but a scratch jordie

**Comment by PKPenguin321** [comment]
this doesnt work the message always displays no matter what

**Pull request merged**